pages/admin/lessons/[lessonSlug]/[pageName]/index.tsx

```diff
@@ -162,7 +161,12 @@ const ModulesPage = ({ modules, lessonId, refetch }: ModulesPageProps) => {
     <div className={styles.container__modulesPanel}>
       <AdminLessonSideNav
         title="Modules"
-        items={modules}
+        items={modules.map(module => ({
```

Comment:
`AdminLessonSideNav ` takes a specific object. For it to work, we're passing what it expects.